lib/Dialect/FIRRTL/InstanceGraph.cpp

```diff
@@ -0,0 +1,32 @@
+//===- InstanceGraph.cpp - Instance Graph -----------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#include "llvm/Support/Debug.h"
+#include "circt/Dialect/FIRRTL/InstanceGraph.h"
```

Comment:
Plz include InstanceGraph.h first.  You can put a blank line between the includes if clang-format isn't cooperating